modules/microshift-install-rpms.adoc

```diff
@@ -20,28 +20,26 @@ Use the following procedure to install {microshift-short} from an RPM package.
 [source,terminal,subs="attributes+"]
 ----
 $ sudo subscription-manager repos \
-    --enable rhocp-<4.18>-for-<9>-$(uname -m)-rpms \ # <1>
-    --enable fast-datapath-for-<9>-$(uname -m)-rpms # <2>
+    --enable rhocp-${ocp-version}-for-rhel-${op-system-version-major}-$(uname -m)-rpms \
+    --enable fast-datapath-for-rhel-${op-system-version-major}-$(uname -m)-rpms # <2>
 ----
-<1> Replace _<4.18>_ and _<9>_ with the compatible versions of your {microshift-short} and {op-system-base-full}.
-<2> Replace  _<9>_ with the compatible version of {op-system-base}.
 
 . For extended support (EUS) releases, also enable the EUS repositories by running the following command:
 +
-[source,terminal]
+[source,terminal,subs="attributes+"]
 ----
-`$ sudo subscription-manager repos \
-    --enable rhel-<9>-for-x86_64-appstream-eus-rpms \ # <1>
-    --enable rhel-<9>-for-x86_64-baseos-eus-rpms` # <2>
+$ sudo subscription-manager repos \
+    --enable rhel-{op-system-version-major}-for-x86_64-appstream-eus-rpms \ # <1>
+    --enable rhel-{op-system-version-major}-for-x86_64-baseos-eus-rpms` # <2>
 ----
-<1> Replace _<9>_ with the compatible major version number of {op-system-base}.
-<2> Replace _<9>_ with the compatible major version number of {op-system-base}.
+<1> Use the compatible major version number of {op-system-base}.
+<2> Use the compatible major version number of {op-system-base}.
 
 . Avoid unintended future updates into an unsupported configuration by locking your operating system version with the following command:
 +
-[source,terminal]
+[source,terminal,subs="+quotes"]
 ----
-$ sudo subscription-manager release --set=<9.4> command. # <1>
+$ sudo subscription-manager release --set=_<9.4>_ command. # <1>
```

Comment:
Do we have a variable for `{op-system-version-minor}`? I would just put it here because there's only one OS per-release we usually support.
In any case, please remove `<>` around 9.4. Let's just have a copiable command with note if we want to leave it this way.

Reply:
This, too, is evergreen. I do not want Docs updating this value every release.

Reply:
Can't we have something like the following?

```
$ sudo subscription-manager release --set={op-system-version-major}.{op-system-version-minor} command. # <1>

1. Replace {op-system-version-major}.{op-system-version-minor} with he major and minor version of your compatible RHEL system.
```

Reply:
Ah, we do have an attribute. We had set this up with attributes way back in 4.12, then later removed.